src/mono/mono/mini/interp/transform.c

```diff
@@ -2910,6 +2910,8 @@ interp_inline_method (TransformData *td, MonoMethod *target_method, MonoMethodHe
 	if (td->verbose_level)
 		g_print ("Inline start method %s.%s\n", m_class_get_name (target_method->klass), target_method->name);
 
+	MONO_PROFILER_RAISE (inline_method, (td->rtm->method, target_method));
```

Comment:
Is it ok to trigger the event here even if the method will end up not being inlined ? Maybe we should do it at the end of inlining, if it was successful ?